avogadro/qtplugins/plugindownloader/downloaderwidget.cpp

```diff
@@ -0,0 +1,242 @@
+#include "downloaderwidget.h"
+#include "zipextracter.h"
+#include "ui_downloaderwidget.h"
+
+namespace Avogadro {
+namespace QtPlugins {
+
+DownloaderWidget::DownloaderWidget(QWidget *parent)
+    : QDialog(parent), ui(new Ui::DownloaderWidget) {
```

Comment:
We can probably just fix this with clang-format, but functions etc have the opening { on their own line